org.eclipse.draw2d/src/org/eclipse/draw2d/geometry/PrecisionRectangle.java

```diff
@@ -26,38 +26,42 @@ public final class PrecisionRectangle extends Rectangle {
 	/**
 	 * Double value for height
 	 *
-	 * @noreference
+	 * @noreference This field is not intended to be referenced by clients.
 	 * @deprecated Use {@link #setPreciseHeight(double)} and
 	 *             {@link #preciseHeight()} instead. This field will become private
 	 *             in the future.
 	 */
+	@Deprecated(since = "3.7", forRemoval = true)
```

Comment:
This field should not be removed but turned private!

Reply:
Of course it's not the field that's removed, but rather the access to said field. What I want to achieve is that users get a "removal" warning, rather than a "deprecation" warning because from experience people tend to treat a deprecation like a "suggestion" and will only adapt their code once it's actually removed.

Sadly, I don't believe the annotation allows for such a level of granularity so I went with this, just to really bring the point across that people should move to the getters and setters instead.

Reply:
I'm stupid the what will happen is anyhow in the Java-doc comment.